desktop/core/src/desktop/lib/fs/ozone/upload.py

```diff
@@ -31,22 +31,23 @@
 else:
   from django.utils.translation import ugettext as _
 
+from desktop.conf import TASK_SERVER_V2
 from desktop.lib.exceptions_renderable import PopupException
-from filebrowser.utils import generate_chunks, calculate_total_size
-from desktop.conf import TASK_SERVER
+from filebrowser.utils import calculate_total_size, generate_chunks
 
 LOG = logging.getLogger()
 
+
 class OFSFineUploaderChunkedUpload(object):
   def __init__(self, request, *args, **kwargs):
     self.qquuid = kwargs.get('qquuid')
     self.qqtotalparts = kwargs.get('qqtotalparts')
     self.totalfilesize = kwargs.get('qqtotalfilesize')
     self.file_name = kwargs.get('qqfilename')
     if self.file_name:
-      self.file_name = unicodedata.normalize('NFC', self.file_name) # Normalize unicode
+      self.file_name = unicodedata.normalize('NFC', self.file_name)  # Normalize unicode
     self.chunk_size = UPLOAD_CHUNK_SIZE.get()
-    if kwargs.get('chunk_size', None) != None:
+    if kwargs.get('chunk_size', None) is not None:
```

Comment:
one can remove "is not None" from the check.

Reply:
Removed this check.